### PR TITLE
Simplify parsing git output in build_bundle.

### LINKED ISSUE #25: Localisation Problem
In  circuitpython-build-tools/circuitpython_build_tools/scripts/build_bundles.py
Line 92: if line.startswith(b"Entering") or not line:

the script fails if your language is different, e.g. de_DE

Fix for my Setting:
 if line.startswith(b"Entering") or line.startswith(b"Betrete") or not line:

But that does not help for any other language. 
Maybe add a hint to set your environment to C or en_US.

### PR DESCRIPTION
**Pull request by dhalbert**
Fixes #25 by suppressing the printing of "Entering ..." or something else in another language.

### CODE REVIEW

**Comment by dhalbert** [comment]
Also fixes #26 with an edit to README.me.

File: README.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ circuitpython-build-tools package.
 ```shell
 python3 -m venv .env
 source .env/bin/activate
+pip install wheel
```
Comment (sommersoft) [inline README.md]: Would it be easier/better to add `wheel` to requirements.txt?

Comment (dhalbert) [reply README.md]: Good idea. Will do that.

**Comment by dhalbert** [comment]
I tested that particular line iwth the `--quiet` manually, but not the whole thing.

**Pull request merged**